webapp/views.py

```diff
@@ -64,7 +64,15 @@ def _build_mirror_list():
         flask.request.headers.get("X-Real-IP", flask.request.remote_addr)
     )
 
-    if ip_location and "country" in ip_location:
+    if not local:
+        for mirror in mirrors:
+            mirror_list.append(
+                {
+                    "link": mirror["link"],
+                    "bandwidth": mirror["mirror_bandwidth"],
+                }
+            )
+    elif ip_location and "country" in ip_location:
         country_code = ip_location["country"]["iso_code"]
 
         for mirror in mirrors:
```

Comment:
The if condition below is getting split on 3 lines. 

I'd extract `mirror["mirror_countrycode"]` as a variable `mirror_countrycode`. 
..and extract `mirror["link"].startswith("https")` as a variable `is_https`.
Rewrite the condition as `if mirror_countrycode == contry_code and is_https:` 

But I leave it up to you.

Reply:
Good comments, I was not changing these from what it was before, but I can clean it up a bit while I'm on it.